tests/ctst/steps/utils/utils.ts

```diff
@@ -196,18 +196,21 @@ async function createBucketWithConfiguration(
 }
 
 async function putObject(world: Zenko, objectName?: string) {
-    world.addToSaved('objectName', objectName || Utils.randomString());
-    const objectNameArray = world.getSaved<string[]>('objectNameArray') || [];
-    objectNameArray.push(world.getSaved<string>('objectName'));
-    world.addToSaved('objectNameArray', objectNameArray);
+    world.resetCommand();
+    let finalObjectName = objectName;
+    if (!finalObjectName) {
+        finalObjectName = `${Utils.randomString()}`;
+    }
+    world.addToSaved('objectName', finalObjectName);
```

Comment:
is it needed? It this will be done by `putObject()` on line 209...

Reply:
The line 209 only calls the S3 API wrapper from cli-testing, that will either run a cli or sdk command, but it is already outside of the world's scope, and it won't change the 'objName', only consume the input parameters...